specs/protocol/isthmus/predeploys.md

```diff
@@ -67,6 +73,7 @@ graph LR
   BaseFeeVault -- "getConfig(ConfigType.GAS_PAYING_TOKEN)(address,uint256,uint8)" --> L1Block
   SequencerFeeVault -- "getConfig(ConfigType.SEQUENCER_FEE_VAULT_CONFIG)(address,uint256,uint8)" --> L1Block
   L1FeeVault -- "getConfig(ConfigType.L1_FEE_VAULT_CONFIG)(address,uint256,uint8)" --> L1Block
+  OperatorFeeVault -- "getConfig(ConfigType.OPERATOR_FEE_VAULT_CONFIG)(address,uint256,uint8)" --> L1Block
```

Comment:
i am confused about this integration into std L2 genesis, will need to look at what the implementation does

Reply:
I detangled the specs, now there are 2 separate sections in configurability.md:
- Fee vault config
- Operator fee parameters